examples/CMakeLists.txt

```diff
@@ -11,6 +11,10 @@ if(protobuf_VERBOSE)
   message(STATUS "Using Protocol Buffers ${Protobuf_VERSION}")
 endif()
 
+if(protobuf_WITH_ZLIB)
+  find_package(ZLIB)

```

Comment:
This is a common issue associated with imported targets and finding dependencies, which I still do not fully understand.
I have suggested that we can put this code in protobuf-config.cmake and relieve the user of this work is. But I do not know the true test of it or not?